phpseclib/File/X509.php

```diff
@@ -3117,12 +3118,13 @@ public function removeExtension($id)
      *
      * @param string $id
      * @param array $cert optional
+     * @param string $path
      * @access public
      * @return mixed
      */
-    public function getExtension($id, $cert = null)
+    public function getExtension($id, $cert = null, $path)
     {
-        return $this->getExtensionHelper($id, $cert);
+        return $this->getExtensionHelper($id, $cert, $path);
```

Comment:
Also, spaces ought to be added before and after the `=` in the `$path=null` bit. PSR-1/2 do appear to discuss this but in areas where those standards do not provide clarity I'd say, as a general rule of thumb, the PEAR standards apply. Quoting https://pear.php.net/manual/en/standards.funcalls.php ,

> As displayed above, there should be one space on either side of an equals sign used to assign the return value of a function to a variable.

idk how well the phpseclib codebase actually conforms to these standards but, anyway, I can take care of this when I merge.